src/transformers/modeling_bert.py

```diff
@@ -1228,13 +1228,14 @@ def forward(
         position_ids=None,
         head_mask=None,
         inputs_embeds=None,
-        next_sentence_label=None,
+        labels=None,
```

Comment:
if model has only one loss as is the case here, I think we should force the name to be `labels`.

Reply:
I agree with that approach. This might require special new dataloaders to change the labels name, but this is easy enough to do.